src/main.js

```diff
@@ -0,0 +1,119 @@
+//  after getting to extension to update rows and discs added the wincondition property and used Object assign to make a copy of the original board so that my board property could be manipulated with moveDisc without affecting the winCondition property
+
+var Game = function () {
+  this.board = [[5, 4, 3, 2, 1], [], []];
+  this.originalBoardState = [[5, 4, 3, 2, 1], [], []];
+  this.displayBoard = function () {
+    var display = this.board.map(function (row) {
+      return "---" + " " + row.join(" ");
+    });
+    display.forEach(function (row) {
+      console.log(row);
+    });
+  };
+  this.moveDisc = function (currentPeg, newPeg) {
+    if (
+      typeof currentPeg === "number" &&
+      typeof newPeg === "number" &&
+      currentPeg <= this.board.length &&
+      currentPeg > 0 &&
```

Comment:
this is a lot to put into a condition - maybe have a helper function called validateMove